commands/Moderation/nickname.js

```diff
@@ -0,0 +1,22 @@
+exports.run = async (client, msg, [user, name = ""]) => {
+  await user.setNickname(name);
```

Comment:
You might want to check if the selected member has higher/equal role position than the author, so the bot doesn't change the nickname to users with a role higher than yours.